homeassistant/components/motioneye/sensor.py

```diff
@@ -0,0 +1,91 @@
+"""Sensor platform for motionEye."""
+from __future__ import annotations
+
+import logging
+from types import MappingProxyType
+from typing import Any
+
+from motioneye_client.client import MotionEyeClient
+from motioneye_client.const import KEY_ACTIONS, KEY_NAME
+
+from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from . import MotionEyeEntity, get_camera_from_cameras, listen_for_new_cameras
+from .const import CONF_CLIENT, CONF_COORDINATOR, DOMAIN, TYPE_MOTIONEYE_ACTION_SENSOR
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up motionEye from a config entry."""
+    entry_data = hass.data[DOMAIN][entry.entry_id]
+
+    @callback
+    def camera_add(camera: dict[str, Any]) -> None:
+        """Add a new motionEye camera."""
+        async_add_entities(
+            [
+                MotionEyeActionSensor(
+                    entry.entry_id,
+                    camera,
+                    entry_data[CONF_CLIENT],
+                    entry_data[CONF_COORDINATOR],
+                    entry.options,
+                )
+            ]
+        )
+
+    listen_for_new_cameras(hass, entry, camera_add)
+
+
+class MotionEyeActionSensor(MotionEyeEntity, SensorEntity):
+    """motionEye action sensor camera."""
+
+    def __init__(
+        self,
+        config_entry_id: str,
+        camera: dict[str, Any],
+        client: MotionEyeClient,
+        coordinator: DataUpdateCoordinator,
+        options: MappingProxyType[str, str],
+    ) -> None:
+        """Initialize an action sensor."""
+        super().__init__(
+            config_entry_id,
+            TYPE_MOTIONEYE_ACTION_SENSOR,
+            camera,
+            client,
+            coordinator,
+            options,
+            SensorEntityDescription(
+                key=TYPE_MOTIONEYE_ACTION_SENSOR, entity_registry_enabled_default=False
+            ),
+        )
+
+    @property
+    def name(self) -> str:
+        """Return the name of the sensor."""
+        camera_prepend = f"{self._camera[KEY_NAME]} " if self._camera else ""
+        return f"{camera_prepend}Actions"
+
+    @property
+    def state(self) -> int:
```

Comment:
```suggestion
    def native_value(self) -> StateType:
```

Reply:
Is there a `native_unit_of_measurement`?


https://developers.home-assistant.io/docs/core/entity/sensor/#properties

Reply:
It's just an absolute count / gauge, so no particular unit would make sense here I suspect.